test/rules/std_test.py

```diff
@@ -76,9 +76,11 @@
         (
             "L016",
             "block_comment_errors.sql",
-            [(1, 121), (2, 99), (4, 88)],
+            # Two errors on line one, because we call out the missing
+            # indent *and* the long line as two issues.
+            [(1, 7), (1, 8), (2, 5), (4, 5)],
         ),
-        ("L016", "block_comment_errors_2.sql", [(1, 85), (2, 86)]),
+        ("L016", "block_comment_errors_2.sql", [(1, 1), (2, 1)]),
```

Comment:
What changed here?

Reply:
Same as previous comment. Long line checks now anchor on the first non-indent char rather than the closing newline.